packages/big-design/src/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,63 @@
+import { ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { useState } from 'react';
+
+import { useUniqueId } from '../../hooks';
+import { typedMemo } from '../../utils';
+import { Box } from '../Box';
+
+import { StyledButton } from './styled';
+
+export interface CollapseProps extends React.HTMLAttributes<HTMLDivElement> {
+  title?: string;
+  initiallyOpen?: boolean;
+  onCollapseChange?(isOpen: boolean): void;
+}
+
+export const Collapse: React.FC<CollapseProps> = typedMemo(
```

Comment:
I don't think you need a `typedMemo` here, we use `typedMemo` instead of `memo` when our component needs a ts generic.

In fact, having a memo in this component is not going to do much since we are rendering `{children}` which will invalidate memoization. So let's not use memo here.

Reply:
Ok, I tested. It has no effect. Now this FC doesn't use `React.memo`.